partiql-eval/src/lib.rs

```diff
@@ -25,16 +28,34 @@ mod tests {
         };
 
         let evaluable = planner.compile(logical);
-
         let mut evaluator = Evaluator::new(bindings, evaluable);
+
         evaluator.execute();
 
         println!("{:?}", &output);
-
         let result = &output.borrow_mut().output;
         result.clone()
     }
 
+    // TODO: rename once we move to DAG model completely
+    fn evaluate_dag(logical: LogicalPlan, bindings: MapBindings<Value>) -> Value {
+        // TODO remove once we agree on using evaluate output in the following PR:
+        // https://github.com/partiql/partiql-lang-rust/pull/202
```

Comment:
Since we decided to move forward with the DAG model, should we remove output now or will it be removed in an upcoming PR?

Reply:
It'll be removed in a subsequent PR.